compiler/ras/CallStack.cpp

```diff
@@ -229,6 +230,11 @@ void TR_LinuxCallStackIterator::printStackBacktrace(TR::Compilation *comp)
       }
    free(symbols);
    }
+#else
+void TR_LinuxCallStackIterator::printStackBacktrace(TR::Compilation *comp)
+	{
+	}
+#endif /* defined(__GLIBC__) */
```

Comment:
I'm ok with this change; just might impact some debugging when stuff crashes, but backtrace doesn't seem to be something musl wants to deal with.